activestorage/lib/active_storage/previewer/video_previewer.rb

```diff
@@ -16,10 +22,10 @@ def preview(**options)
 
     private
       def draw_relevant_frame_from(file, &block)
-        draw ffmpeg_path, "-i", file.path, "-y", "-vframes", "1", "-f", "image2", "-", &block
+        draw self.class.ffmpeg_path, "-i", file.path, "-y", "-vframes", "1", "-f", "image2", "-", &block
       end
 
-      def ffmpeg_path
+      def self.ffmpeg_path
         ActiveStorage.paths[:ffmpeg] || "ffmpeg"
       end
```

Comment:
Move this up with the other class methods. We prefer to use a `class << self` block when there’s more than one, so switch to that:

```ruby
class << self
  def accept?(blob)
    # ...
  end

  def ffmpeg_exists?
    # ...
  end

  def ffmpeg_path
    # ...
  end
end
```

Reply:
Thanks, @georgeclaghorn . I did the necessary changes and tested it on local. Please have a look.